sdk/hybridcompute/Azure.ResourceManager.HybridCompute/src/Generated/HybridComputeMachineData.cs

```diff
@@ -32,21 +37,135 @@ public HybridComputeMachineData(AzureLocation location) : base(location)
+            OSName = osName;
+            OSVersion = osVersion;
+            OSType = osType;
+            VmUuid = vmUuid;
+            Extensions = extensions;
+            OSSku = osSku;
+            DomainName = domainName;
+            AdFqdn = adFqdn;
+            DnsFqdn = dnsFqdn;
+            PrivateLinkScopeResourceId = privateLinkScopeResourceId;
+            ParentClusterResourceId = parentClusterResourceId;
+            MssqlDiscovered = mssqlDiscovered;
+            DetectedProperties = detectedProperties;
         }
 
-        /// <summary> Hybrid Compute Machine properties. </summary>
-        public MachineProperties Properties { get; set; }
         /// <summary> The list of extensions affiliated to the machine. </summary>
         public IReadOnlyList<HybridComputeMachineExtensionData> Resources { get; }
         /// <summary> Identity for the resource. Current supported identity types: SystemAssigned. </summary>
         public ManagedServiceIdentity Identity { get; set; }
+        /// <summary> Metadata pertaining to the geographic location of the resource. </summary>
+        public LocationData LocationData { get; set; }
+        /// <summary> Configurable properties that the user can set locally via the azcmagent config command, or remotely via ARM. </summary>
+        public AgentConfiguration AgentConfiguration { get; }
+        /// <summary> Statuses of dependent services that are reported back to ARM. </summary>
+        public HybridComputeServiceStatuses ServiceStatuses { get; set; }
+        /// <summary> The metadata of the cloud environment (Azure/GCP/AWS/OCI...). </summary>
+        internal CloudMetadata CloudMetadata { get; set; }
+        /// <summary> Specifies the cloud provider (Azure/AWS/GCP...). </summary>
+        public string CloudMetadataProvider
+        {
+            get => CloudMetadata is null ? default : CloudMetadata.Provider;
+        }
+
+        /// <summary> The info of the machine w.r.t Agent Upgrade. </summary>
+        public AgentUpgrade AgentUpgrade { get; set; }
+        /// <summary> Specifies the operating system settings for the hybrid machine. </summary>
+        public OSProfile OSProfile { get; set; }
+        /// <summary> The provisioning state, which only appears in the response. </summary>
+        public string ProvisioningState { get; }
+        /// <summary> The status of the hybrid machine agent. </summary>
+        public HybridComputeStatusType? Status { get; }
+        /// <summary> The time of the last status change. </summary>
+        public DateTimeOffset? LastStatusChange { get; }
+        /// <summary> Details about the error state. </summary>
+        public IReadOnlyList<ResponseError> ErrorDetails { get; }
+        /// <summary> The hybrid machine agent full version. </summary>
+        public string AgentVersion { get; }
+        /// <summary> Specifies the hybrid machine unique ID. </summary>
+        public string VmId { get; set; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#9f6fc0ae7c8e497e98d7979dadf97a4c).

Reply:
changed the type of VmId to Guid.